docs/docs/gatsby-on-linux.md

```diff
@@ -4,7 +4,9 @@ title: Gatsby on Linux
 
 # Linux
 
-TODO
+> This is a TODO. Help our community expand it.
```

Comment:
Is this _still_ a TODO?

Reply:
The Linux part: Yes. I didn’t find instructions for „real“ Linux